src/LayoutManager/LayoutManager.ts

```diff
@@ -259,10 +259,8 @@ export class LayoutManager {
     { offset }: Required<LayoutResult>,
     object: FabricObject
   ) {
-    object.set({
-      left: object.left + offset.x,
-      top: object.top + offset.y,
-    });
+    object.left += offset.x;
+    object.top += offset.y;
```

Comment:
This may seem to you the same but it may not be
`set` invalidates the cache
If the group is nested and the layout doesn't bubble I think you will hit the case you need it
Am I right?

Reply:
So why do we have specific set dirty in the layout manager? Doesn't that cover all?

Reply:
Correct